tests/test_operator_complex.cpp

```diff
@@ -263,12 +263,12 @@ test_op_assign("Test complex assign division cplx-cplx overload", "[div]", /=);
     sycl::free(cplx_inout, Q);                                                 \
   }
 
-test_op_assign("Test complex assign addition cplx-deci overload", "[add]", +=);
```

Comment:
Why some tests are disable?

Reply:
Because I accidentally committed I file I didn't want to :P...

Reply:
should be fixed now. Just github action config, no source changes.